PIL/ImageMode.py

```diff
@@ -40,6 +40,7 @@ def getmode(mode):
             _modes[m] = ModeDescriptor(m, bands, basemode, basetype)
         # extra experimental modes
         _modes["LA"] = ModeDescriptor("LA", ("L", "A"), "L", "L")
+        _modes["La"] = ModeDescriptor("La", ("L", "a"), "L", "L")
```

Comment:
@wiredfool If you don't mind about adding `La` in `_modes`, I'd like to merge it. I don't see any other controversial points in this RP.